### ISSUE TITLE
Cell formatting for invalid cell input

### ISSUE THREAD

**Issue by RacheleMoresi** [AllenFang/react-bootstrap-table#830]
Any chance of exposing more than just the cell value to the validator function or setting a Bootstrap danger class to the input when the toastr message fires for failed cell validation?

**Comment by AllenFang** [comment]
hi @RacheleMoresi, ```toastr``` is the default message displayer, how about I pass the result of validation by a callback function for you ?  maybe like following:

```js
afterValidate(result) {
   //...
} 

render() {
    const options ={
        disableToastr: true,
        afterValidate: this.afterValidate
    };
    return (
        <BootstrapTable data={products} options={options} ...>
    );
}
```

**Comment by RacheleMoresi** [comment]
I've got a custom error panel now, and am hiding the toastr with css, but the disableToastr option would be really helpful, I think. 

Thanks!

**Comment by AllenFang** [comment]
@RacheleMoresi, did you use the cell editing? or the insert modal for a new row(```insertRow```)?

**Comment by RacheleMoresi** [comment]
Cell editing.

**Comment by AllenFang** [comment]
ok, I'll toward https://github.com/AllenFang/react-bootstrap-table/issues/830#issuecomment-260944336 to support the validation fail without ```Toastr```, but it need some time. Thanks

**Comment by AllenFang** [comment]
@RacheleMoresi, fixed on ```v2.9.0```, check https://github.com/AllenFang/react-bootstrap-table/commit/7ea2d9532d7d2d2a4d6464324193000616da78c5 for example.

If you want to avoid toastr, add ```options.beforeShowError``` and return `false` always
